packages/core/src/plugins/use-masked-errors.ts

```diff
@@ -47,6 +52,16 @@ export type UseMaskedErrorsOpts = {
    * The default value is `process.env['NODE_ENV'] === 'development'`
    */
   isDev?: boolean;
+  /**
+   * Whether parse errors should be masked.
+   * @default false
+   */
+  onParse?: boolean;
+  /**
+   * Whether validation errors should be masked.
+   * @default false
+   */
+  onValidate?: boolean;
```

Comment:
I don't like the name of this options:

Any alternatives? `handleParseErrors`, `handleValidateErrors`?